udata_recommendations/templates/dataset-recommendations.html

```diff
@@ -0,0 +1,26 @@
+<style type="text/css">
+#dataset-recommendations .dataset-card {
+    margin: 0 0 1em 0 !important;
+}
+#dataset-recommendations .recommendation {
+    margin-bottom: 1em;
+}
+#dataset-recommendations .recommendation > * {
+    height: 100%;
+}
+</style>
+
+<!-- dataset recommendations -->
+<div id="dataset-recommendations" class="hidden-xs">
+    <h3>{{ _('See also') }}</h3>
+    <div id="dataset-recommendations-container" class="card-list card-list--columned">
+        {% for reco in recommendations %}
+            <div data-track-content data-content-name="dataset recommendations" data-content-piece="reco {{ loop.idx }}"
+                 data-content-target="datasets/{{ reco }}" data-udata-dataset="{{ reco }}"
+                 class="recommendation col-xs-12 col-md-6">
+            </div>
+        {% endfor %}
+    </div>
+</div>
+
+<script src="/static/oembed.js" type="text/javascript"></script>
```

Comment:
I think now this is only server side, we can simply render the dataset card instead of using the oembed:
- less script to transfer
- less style to transfer
- less requests to perform

Reply:
Of course! 🤦‍♂️